src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -40,7 +41,10 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId }) => {
   const dispatch: any = useDispatch();
 
   const videoWrapper = useRef<HTMLDivElement>(null);
-
+  const { width } = useWindowDimensions();
+  const extremeSmallScreenBreakpoint = 320;
+  const isExtremeSmallScreen =
+    width <= extremeSmallScreenBreakpoint ? true : false;
```

Comment:
In our tailwind config, we have a `vs:` modifier configured for very small screen sizes. Use that instead of introducing javascript code to handle it.

<img width="581" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/3e8fa4ec-d3d0-4f55-8cab-d574ec192007">